portfolio/banking/banking_types.go

```diff
@@ -0,0 +1,44 @@
+package banking
+
+import (
+	"sync"
+)
+
+const (
+	// ErrBankAccountNotFound message to return when bank account was not found
+	ErrBankAccountNotFound = "bank account ID: %v not found"
+	// ErrAccountCannotBeNil message to return when bank account is nil
+	ErrAccountCannotBeNil = "Account cannot be nil"
+	// ErrAccountCannotBeEmpty message to return when bank account number is empty
+	ErrAccountCannotBeEmpty = "Bank Account Number cannot be empty"
+	// ErrBankAccountDisabled message to return when bank account is disabled
+	ErrBankAccountDisabled = "Bank Account is disabled"
+	// ErrBSBRequiredforAUD message to return when currency is AUD but no bsb is set
+	ErrBSBRequiredforAUD = "BSB must be set for AUD values"
+	// ErrIBANSwiftNotSet message to return when no iban or swift value set
+	ErrIBANSwiftNotSet = "IBAN/SWIFT values not set"
+)
+
+// Account holds differing bank account details by supported funding
+// currency
+type Account struct {
+	Enabled             bool    `json:"enabled"`
+	ID                  string  `json:"id,omitempty"`
```

Comment:
It would be good to fill the ID field in the config-example.json since we use it in the CLI